atc/creds/ssm/ssm_test.go

```diff
@@ -164,6 +164,19 @@ var _ = Describe("Ssm", func() {
 			Expect(err).To(BeNil())
 		})
 
+		It("should get shared parameter if exists", func() {
```

Comment:
do you think there's merit to adding a test for the opposite case, ie: the shared parameter doesn't exist?

Reply:
Rui and i discussed this synchronously, and i wrote up the summary here: https://github.com/concourse/concourse/issues/8897

Gonna close this review now and we'll figure out the tests later.